src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetPackageSearchService.cs

```diff
@@ -493,17 +492,28 @@ private async ValueTask<IInstalledAndTransitivePackages> GetInstalledAndTransiti
             {
                 if (isSolution)
                 {
+                    // We only need the installed packages? What about recommender
```

Comment:
The comment "What about recommender" looks like a TODO.

Reply:
This was a comment to explain which data we need for each PM UI tab in both Project and Solution View